api/v1alpha1/workflow_webhook.go

```diff
@@ -116,28 +123,30 @@ func (w *Workflow) ValidateUpdate(old runtime.Object) error {
 		return nil
 	}
 
+	statusPath := field.NewPath("Status")
+
 	// Validate the elements in the Drivers array
 	for i, driverStatus := range w.Status.Drivers {
+
```

Comment:
Could/Would there ever be multiple Drivers? If possible, Child gets a little tricky here since it's an array index; I'm not sure there's a convention but we could maybe use jq style?
`field.NewPath("Status").Child("Drivers").Child(fmt.Sprintf("[%d]", i).Child("Completed")`

Reply:
Oh, there's a `Path.Index()` for this https://github.com/NearNodeFlash/lustre-fs-operator/blob/f537650db1c620ffdf74f76b3aac3a4d18b1108c/vendor/k8s.io/apimachinery/pkg/util/validation/field/path.go#L79-L83